host-interaction/cli/resolve-path-using-msvcrt.yml

```diff
@@ -0,0 +1,11 @@
+rule:
+  meta:
+    name: resolve path using msvcrt
+    namespace: host-interaction/cli
+    author: "@_re_fox"
+    scope: basic block
+    examples:
+      - 31600ad0d1a7ea615690df111ae36c73:0x4016B8
+  features:
+    - or:
+      - api: msvcrt.__p__pgmptr
```

Comment:
let's add wpgmptr, _get_pgmptr and _get_wpgmptr?

see https://docs.microsoft.com/en-us/cpp/c-runtime-library/pgmptr-wpgmptr?view=vs-2019

Reply:
Yep, thanks for the spot.  I found the following ones crawling the exports.
```
   d8    35e40  __p__pgmptr
   e1    35ed0  __p__wpgmptr
  1c4    66310  _get_pgmptr
  1ca    66440  _get_wpgmptr
  316    b6074  _pgmptr
  460    b6078  _wpgmptr
```
I'll add those in.

Reply:
awesome, thanks a lot!